### DISCUSSION TITLE
Ability to change the wording of  # of "Books" -> "Chapters" for Manga

### DISCUSSION THREAD

**Discussion by zolfey** [gotson/komga#1711] [Ideas]
It may already be possible, but I couldn't find anything about it in the documentation. I would like the ability to change what the text says underneath of the comic/manga depending on what it is. 

If this is possible sorry it is nitpicky I know but something I would like to see.

![image](https://github.com/user-attachments/assets/c9a412de-a281-4216-8656-c0048c9c2c17)

**Comment by gotson** [comment]
this is not possible nor something i want to add in Komga
